Sources/Testing/Running/EntryPoint.swift

```diff
@@ -10,22 +10,120 @@
 
 private import TestingInternals
 
+#if canImport(Foundation) && !SWT_NO_ABI_ENTRY_POINT
+/// The type of the entry point to the testing library used by tools that want
+/// to remain version-agnostic regarding the testing library.
+///
+/// - Parameters:
+///   - argumentsJSON: A buffer to memory representing the JSON encoding of an
+///     instance of `__CommandLineArguments_v0`. If `nil`, a new instance is
+///     created from the command-line arguments to the current process.
+///   - eventHandler: An event handler to which is passed a buffer to memory
+///     representing each event and its context, as with ``Event/Handler``, but
+///     encoded as JSON.
```

Comment:
Should this comment be amended given the recent change to only pass a single argument to the callback?

Reply:
I tried to phrase the comment correctly for a single argument, i.e. "**a** buffer to memory representing each event and its context" but I can rephrase it if you have a suggestion.